src/order-book/api.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Address,
+  BaseHttpRequest,
+  CancelablePromise,
+  DefaultService,
+  OpenAPIConfig,
+  OrderBookClient,
+  OrderCancellation,
+  OrderCreation,
+  OrderPostError,
+  OrderQuoteRequest,
+  OrderQuoteResponse,
+  Trade,
+  TransactionHash,
+  UID,
+} from './generated'
+import { CowError } from '../common/cow-error'
+import { SupportedChainId } from '../common/chains'
+import { EnvConfig, PROD_CONFIG, STAGING_CONFIG } from '../common/configs'
+import { transformOrder } from './transformOrder'
+import { EnrichedOrder } from './types'
+import { ApiRequestOptions } from './generated/core/ApiRequestOptions'
+import { request as __request } from './generated/core/request'
```

Comment:
Why is this generated? Wasn't this `crossFetch` before?

Reply:
It's a part of `openapi-typescript-codegen` result (autogenerated code).
We use it here just to customize headers ('Content-Type')